solver/llbsolver/proc/provenance.go

```diff
@@ -68,6 +66,11 @@ func ProvenanceProcessor(attrs map[string]string) llbsolver.Processor {
 			}
 		}
 
+		var inlineOnly bool
+		if v, err := strconv.ParseBool(attrs["inline-only"]); v && err == nil {
```

Comment:
I'm not a fan of `inline-only` as the name of the option in hindsight. 1. It has a different meaning to `inline` for the exporters, and 2. it's not clear what property makes image/oci `inline` but not local/tar.

Other ideas welcome here :tada:

Reply:
> I'm not a fan of inline-only

`attach` / `embed` maybe?